SuccessBonusActivationActivity/handler.ts

```diff
@@ -41,13 +41,8 @@ export const UnhandledFailure = t.interface({
 });
 export type UnhandledFailure = t.TypeOf<typeof UnhandledFailure>;
 
-export const TransientFailure = t.interface({
-  kind: t.literal("TRANSIENT")
-});
-export type TransientFailure = t.TypeOf<typeof TransientFailure>;
-
 const SuccessBonusActivationFailure = t.union(
-  [InvalidInputFailure, UnhandledFailure, TransientFailure],
+  [InvalidInputFailure, UnhandledFailure],
```

Comment:
why this change here?

Reply:
`TransientFailure` never happens inside this activity, all errors are remapped to `UnhandledFailure`. The Input decode error is instead `InvalidInputFailure`

Reply:
if an error cause a retry it's better to call it `TransientFailure` so we should rename this (as it's handled actually : )

Reply:
Fixed in 790c445